metadata-models/src/main/pegasus/com/linkedin/common/ContainerPath.pdl

```diff
@@ -0,0 +1,25 @@
+namespace com.linkedin.common
+
+/**
+ * Shared aspect containing Container Paths to be indexed for an entity.
+ */
+@Aspect = {
+    "name": "containerPath"
+}
+record ContainerPath {
+  /**
+   * A valid container path for the entity. This field is provided by DataHub by default.
+   *
+   * Container paths are stored in elasticsearch as slash-separated strings and only include platform specific folders or containers.
+   * These paths should not include high level info captured elsewhere ie. Platform and Environment.
+   */
+   @Searchable = {
+    "/*/id": {
+      "fieldName": "containerPath",
+      "fieldType": "TEXT",
+      "aggregate": true,
+      "delimiter": "/"
+    }
+  }
+  path: array[ContainerPathEntry]
```

Comment:
I know we discussed this previously, but are we _SURE_ that this modeling restriction won't get us into trouble?

Reply:
Is there any world in which it might?

Reply:
hmm @jjoyce0510 do you have anything in mind? i'm definitely going to keep thinking about it before merging... and get approvals from at least one of the ingestion folks